### ISSUE TITLE
Manic Panic Ghosts second player doesn't work

### ISSUE THREAD

**Issue by SuperDavid2** [p1pkin/demul#481]
In Manic Panic Ghosts, there is no way to use the wand as the second player. It's worth noting that it is possible to get the second player working on Demul with the program Demulshooter. The author of that program said this about the game. 
"Well, with a bit of digging it turned out Demul was nulling all P2 values of pokasuka/manicpnc on purpose  :(
That's why the regular hack was not working with them..."

**Comment by p1pkin** [comment]
right, currently P2 inputs is disabled, because it interfere with P1 inputs in current implementation.
